Configuration/AlCa/python/autoCond.py

```diff
@@ -66,9 +66,9 @@
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2021
     'phase1_2021_design'       : '106X_upgrade2021_design_v3', # GT containing design conditions for Phase1 2021
```

Comment:
shouldn't there also be an update for phase1_2021_design or is this GT filled with some other independent/trivial conditions?

Reply:
@slava77, at the moment there is no need to update the 2021 design GT, since conditions there are "trivial" -  in the sense that we don't simulate any radiation damage. At any rate @tsusa has run the resolution analysis using Single Muon pT=10 gun, in the following setup:
```
cmsDriver.py -s GEN,SIM,DIGI,L1,DIGI2RAW,RAW2DIGI,RECO --evt_type SingleMuPt10_pythia8_cfi --fileout file:GENSIMRECO_MuPt10.root --python_filename=Run3_2021_generic_IBC.py -n 10 --no_exec --era Run3 --beamspot  GaussSigmaZ4cm --nThreads=4 —customise_commands="process.mix.digitizers.pixel.ThresholdInElectrons_BPix_L1 = cms.double(1300.0);  process.mix.digitizers.pixel.ThresholdInElectrons_BPix_L2 = cms.double(1600.0);process.mix.digitizers.pixel.ThresholdInElectrons_BPix = cms.double(1600.0); process.mix.digitizers.pixel.ThresholdInElectrons_FPix = cms.double(1600.0);process.PixelCPEGenericESProducer.IrradiationBiasCorrection = True”  --conditions auto:phase1_2021_design
```

and we don't observe significant deviations from the current performance. You can find the results of the test [here](https://cernbox.cern.ch/index.php/s/LP1Tu9Kg30xVG2l).